src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -1047,13 +1066,11 @@ private bool IsSingleBoolFalseOrNullValue(AttributeValue[] values)
             return false;
         }
 
-        private bool UseAttributeNameAsValue(AttributeValue[] values)
+        private bool UseAttributeNameAsValue(string prefix, object value)
```

Comment:
Should change name to reflect new behavior since we're not consuming all of the attributes anymore (how many there are controls if we use the attribute name as value).